package.json

```
@@ -1,13 +1,13 @@
 {
   "author": "Remy Sharp",
   "name": "@freecodecamp/loop-protect",
-  "description": "Prevent infinite loops in dynamically eval'd JavaScript.",
+  "description": "Prevent infinite loops and recursion in dynamically eval'd JavaScript.",
   "main": "dist/",
-  "version": "2.2.1",
-  "homepage": "https://github.com/ojeytonwilliams/loop-protect",
+  "version": "3.0.1",
```

```suggestion
  "version": "2.3.0",
```
Nitpicky, I know, but this is backwards compatible isn't it?